build-local.sh

```diff
@@ -4,5 +4,5 @@ set -e
 
 cd "$(dirname "$0")"
 
-mvn -DskipTests clean package
+mvn -DskipITs clean verify
```

Comment:
Does it mean we will now have unit tests run as part of local build ?

Reply:
yes